internal/flypg/zombie.go

```diff
@@ -117,7 +117,8 @@ func TakeDNASample(ctx context.Context, node *Node, standbys []Member) (*DNASamp
 		sample.totalActive++
 
 		// Record conflict when primary doesn't match.
-		if primary.Hostname != node.MachineID && primary.Hostname != node.PrivateIP {
+		// We're checking PrivateIP here for backwards compatibility
```

Comment:
Do we need to remain backwards compatible here?

Reply:
I think so, during `fly image update`, when we do a rolling upgrade of the nodes